# IDL struct ::niut::HUMAN_LIST
int32 watch_dog
int32 num
niut_USER_STR[16] users
niut_USER_STR[16] filtered_users
